Essentials/src/com/earth2me/essentials/commands/Commandpaytoggle.java

```diff
@@ -15,17 +15,48 @@ public Commandpaytoggle() {
 
     @Override
     public void run(Server server, User user, String commandLabel, String[] args) throws Exception {
-        boolean acceptingPay = !user.isAcceptingPay();
-        if (commandLabel.contains("payon")) {
-            acceptingPay = true;
-        } else if (commandLabel.contains("payoff")) {
-            acceptingPay = false;
-        }
-        user.setAcceptingPay(acceptingPay);
-        if (acceptingPay) {
-            user.sendMessage(tl("payToggleOn"));
+        if (args.length < 1) {
+            boolean acceptingPay = !user.isAcceptingPay();
+            if (commandLabel.contains("payon")) {
+                acceptingPay = true;
+            } else if (commandLabel.contains("payoff")) {
+                acceptingPay = false;
+            }
+            user.setAcceptingPay(acceptingPay);
+            if (acceptingPay) {
+                user.sendMessage(tl("payToggleOn"));
+            } else {
+                user.sendMessage(tl("payToggleOff"));
+            }
         } else {
-            user.sendMessage(tl("payToggleOff"));
+            if (args[0].equalsIgnoreCase("players")) {
```

Comment:
I don't see why this list feature is necessary. Why not just tell the player when the payment is going through that it's blocked?